azure-pipelines.yml

```diff
@@ -77,13 +77,17 @@ stages:
           - name: _BuildArgs
             value: /p:SignType=$(_SignType)
         steps:
+        - task: UseDotNet@2
+          displayName: Install SDK 9.0.102
+          inputs:
+            packageType: sdk
+            version: 9.0.102
```

Comment:
We might want to put this in the linux leg below as well. Would be easier if it was part of a common task list we can stick in all the jobs for all the other OSs.